GomokuClient/packages/gomoku-core/src/hooks/useGameSession.ts

```diff
@@ -37,6 +51,7 @@ export const useJoinGame = (gameID: string) => {
         onPlayerMadeMove: ({ playerId, tile, placedTileColor }) => {
           console.log("Player made move:", playerId, tile, placedTileColor);
           addTile(tile.y, tile.x, placedTileColor as TileColor);
+          setLastTile({ x: tile.x, y: tile.y });
```

Comment:
useEffect has a missing dependency: 'setLastTile'. Either include it or remove the dependency array